test/parallel/test-timers-setunreftimeout.js

```diff
@@ -0,0 +1,58 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+
+const { strictEqual } = require('assert');
+const { setUnrefTimeout } = require('internal/timers');
+
+{
+  common.expectsError(
+    () => setUnrefTimeout(),
+    {
+      code: 'ERR_INVALID_CALLBACK',
+      type: TypeError,
+      message: 'Callback must be a function'
+    }
+  );
+}
+
+// Test this if correctly passes arguments to the callback
+{
+  const maxArgsNum = 4;
+  for (let i = 0; i < maxArgsNum; i++) {
+    const results = [];
+    const inputArgs = [];
+    // set the input argument params
+    for (let j = 0; j <= i; j++) {
+      inputArgs.push(j);
+    }
+
+    // For checking the arguments.length,
+    // callback function should not be arrow fucntion.
```

Comment:
I don't think this comment is relevant now.

Reply:
I agree with you. I'll delete it